frontend/src/app/menu/menu.component.html

```diff
@@ -6,7 +6,7 @@
   <button type="button" class="navbar-toggler" (click)="toggleNavbar()">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <div id="navbar" class="navbar-collapse" [class.collapse]="navbarCollapsed">
+  <div id="navbar" class="collapse navbar-collapse" [ngbCollapse]="navbarCollapsed">
```

Comment:
This is nice to have: we should add it in the pro pack exercise that introduces ng-bootstrap

Reply:
Yes. And of course, since our pro pack is full of accordions, it would be nice to use the latest version there too: we should benefit from the animations without any work.

Note that I asked if they planned to make the `ngbRunTransition` function public, and they should do it (maybe not immediately). And anyway, it's not that long, so we could copy and paste it. I find it nice to apply some custom transitions, without relying on the angular animations module. And the beauty of it is that when animations are disabled, it's purely synchronous.
Not sure if it would be possible to have the same thing for css animations and not transitions.